parsers/c/c_lexer.mll

```diff
@@ -568,9 +605,7 @@ and initial flags = parse
         Hashtbl.find lexicon id
       with Not_found ->
         if flags.inside_cn then
-          try Hashtbl.find cn_lexicon id
-          with Not_found ->
-            LNAME id
```

Comment:
The LNAME is getting lost: cn_lex_keyword is always returning a UNAME in the ultimate Not_found case, so it probably has to parameterised what it returns then. Perhaps related to the `bool` problem?

(UNAME: uppercase name, LNAME: lowercase name)

Reply:
Fixed